addons/mail/static/src/core/common/thread.js

```diff
@@ -56,6 +57,8 @@ export class Thread extends Component {
             showJumpPresent: false,
         });
         this.threadService = useState(useService("mail.thread"));
+        this.personaService = useState(useService("mail.persona"));
```

Comment:
`personaService` looks useless _(was probably used when insert had to be done on the appropriate service, which is no longer the case)_